Fix/commands

jbarnes850 opened the pull request:
# Landing Page Updates and Documentation Alignment

## Overview
This PR updates the landing page and documentation to accurately reflect the current state of FIL-Frame, fixes several UI/UX issues, and improves SEO optimization.

## Key Changes

### Installation Instructions
- Updated CLI commands to match `create-filecoin-app` repository
- Simplified installation steps in both UI and README
- Added specific integration flags documentation
- Fixed Quick Start button to properly scroll to installation section

### SEO and Metadata
- Added comprehensive OpenGraph and Twitter card metadata
- Improved page descriptions and keywords
- Added proper alt text for images
- Added new OG image for social media sharing

### UI/UX Improvements
- Updated navigation to link directly to FIL-Frame GitHub README
- Added "Coming Soon" states for unreleased integrations
- Improved storage/retrieval section clarity
- Fixed documentation links to point to correct Filecoin docs sections

### Documentation Links
Fixed broken links:
- FVM: `https://docs.filecoin.io/smart-contracts/fundamentals/the-fvm`
- Smart Contracts: `https://docs.filecoin.io/smart-contracts/fundamentals`
- Proofs: `https://docs.filecoin.io/basics/the-blockchain/proofs`

### Integration Templates
- Updated section title to "Pre-configured Storage Quickstarts"
- Added proper links to template branches:
  - Lighthouse: `/tree/lighthouse`
  - Storacha: `/tree/storacha-nfts`
- Marked Akave, Axelar, and Lit Protocol as "Coming Soon"

### Technical Updates
- Fixed image configuration warning by updating to `remotePatterns`
- Improved TypeScript types for integration cards
- Added proper scroll behavior for navigation

## Testing
- Verified all installation commands work as expected
- Tested navigation and scroll behavior
- Confirmed all external links are correct
- Validated SEO meta tags using social media preview tools

## Related Issues
Closes #2

xBalbinus commented:
A few things to fix:
- You should link to npx create-filecoin-app instead
- Still need to pull from the FIL-B website for logos.
- All the branches should be linked:
    - Akave: https://github.com/FIL-Builders/fil-frame/tree/akave-integration
    - Axelar: https://github.com/FIL-Builders/fil-frame/tree/axelar-integration
    - Lighthouse: https://github.com/FIL-Builders/fil-frame/tree/lighthouse-nfts
    - Storacha: https://github.com/FIL-Builders/fil-frame/tree/storacha-nfts
    - Data Depot: https://github.com/FIL-Builders/fil-frame/tree/main
- Remove developer survey
<img width="552" alt="Screenshot 2024-11-13 at 3 32 07 PM" src="https://github.com/user-attachments/assets/537d8981-3094-43e8-a127-b4f2af80f2db">
- Fix the favicon (I think you can also take this from FIL-B, lmk if you need it in a file)

xBalbinus commented:
Awesome stuff thus far, make sure to resolve merge conflicts.

jbarnes850 commented:
@xBalbinus Thank you for the feedback, see the revisions here:

## Changes
- Updated CLI installation commands to match create-filecoin-app repository
- Fixed integration template links to point to correct branches
- Added proper SEO metadata and favicon
- Removed developer survey component

## Testing
- Checked all integration links
- Tested page load and SEO meta tags
- Confirmed favicon displays correctly
